src/compute/impl/NextPrimeCompute.java

```diff
@@ -12,8 +12,22 @@ public class NextPrimeCompute implements ICompute {
 	public long compute(int i) {
 		// If our starting value is prime, we need to start from the next integer, as
 		// we're looking for the next greatest prime.
+		/**
+  		* @author Gavin Bell
+    		* validation test
+      		*/
+		boolean vtest=false;
+		while(vtest=false){
+			if(isInteger(i)==true){
+				vtest=true;
+			} else {
+				System.out.println("Invalid: Not an integer");
+			}
 		return isPrimeImproved(i) ? recursiveCompute(i + 1) : recursiveCompute(i);
 	}
```

Comment:
The while loop needs another end bracket after the else block. This is actually why Gradle fails to compile the code.